### PR TITLE
wip: build-env: add a simplified build container

### PR DESCRIPTION
**Pull request by egernst**
Adding a simplified container for *just* building Zephyr. To facilitate this,
created subdirectories for a test-env (original container) and build-env (new
container).

As a next step, the original container should be use the build-env container as
its base image. Putting this up for initial feedback before investing more time on this second part.

Signed-off-by: Eric Ernst <eric@amperecomputing.com>

### CODE REVIEW

**Comment by egernst** [comment]
This PR is addresses https://github.com/zephyrproject-rtos/docker-image/issues/30

File: build-env/install-prereqs.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+# install prerequisite packages:
+dpkg --add-architecture i386
+
+apt-get update && apt-get upgrade -y
+
+DEBIAN_FRONTEND=noniteractive apt-get install -y --no-install-recommends \
+    ca-certificates \
+    ccache \
+    cmake \
```
Comment (aelray) [inline build-env/install-prereqs.sh:17]: why install cmake here when you later on get it directly from kitware?

Comment (egernst) [reply build-env/install-prereqs.sh:17]: IIRC, I was scaling back the already existing tooling which may have had it?  

In any event, I wasn't aware what's installed from kitware.

**Comment by nashif** [comment]
stale

**Pull request closed without merging**